Deeploy/Targets/Generic/Templates/FloatPowTemplate.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-FileCopyrightText: 2025 ETH Zurich and University of Bologna
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, List, Tuple
+
+import numpy as np
+
+from Deeploy.DeeployTypes import NetworkContext, NodeTemplate, OperatorRepresentation
+
+
+class _PowTemplate(NodeTemplate):
+
+    def alignToContext(self, ctxt: NetworkContext,
+                       operatorRepresentation: OperatorRepresentation) -> Tuple[NetworkContext, Dict, List[str]]:
+
+        # Get input and output tensors
+        data_in = ctxt.lookup(operatorRepresentation['data_in'])
+        data_out = ctxt.lookup(operatorRepresentation['data_out'])
+
+        # Get data type (fp32 or fp16)
+        data_type = data_in._type.typeName
+        operatorRepresentation['data_type'] = data_type
+
+        # Exponent must be a constant integer
+        if 'exponent' in operatorRepresentation:
+            exponent_input = operatorRepresentation['exponent']
+            if isinstance(exponent_input, str):
+                # It's a tensor name - not supported for integer exponent version
+                raise ValueError("Tensor exponent not supported. Use constant integer exponent.")
+            else:
+                # Convert to integer
```

Comment:
According to the [official ONNX documentation](https://onnx.ai/onnx/operators/onnx__Pow.html), the Pow node also supports float exponents. Why did you decide to only allow integer ones?